From 246990835855bd3970a7cf8eb0abb667ae0749d3 Mon Sep 17 00:00:00 2001
From: Patrick McCarty <patrick.mccarty@intel.com>
Date: Thu, 14 Jun 2018 12:02:55 -0700
Subject: [PATCH] Add custom vlock PAM config

To avoid the default PAM warn/deny behavior, we need a custom PAM config
for vlock. It will be installed manually in the spec %install section.

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 src/vlock/vlock.pam | 11 +++++++++++
 1 file changed, 11 insertions(+)
 create mode 100644 src/vlock/vlock.pam

diff --git a/src/vlock/vlock.pam b/src/vlock/vlock.pam
new file mode 100644
index 0000000..5683631
--- /dev/null
+++ b/src/vlock/vlock.pam
@@ -0,0 +1,11 @@
+# derived from the LFS gdm-password.pam in gdm
+
+auth     requisite      pam_nologin.so
+auth     required       pam_env.so
+auth     include        system-auth
+
+account  include        common-account
+password include        common-password
+
+session  required       pam_limits.so
+session  include        common-session
-- 
2.17.1

